packages/components/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "target": "esnext",
-        "module": "esnext",
+        "module": "commonjs",
```

Comment:
What's the reason for this change?

Reply:
Trying to get eslint to stop complaining about types